compiler/ml/typecore.ml

```diff
@@ -2256,6 +2256,17 @@ let rec type_exp ~context ?recarg env sexp =
 *)
 
 and type_expect ~context ?in_function ?recarg env sexp ty_expected =
+  (* Special errors for braced identifiers passed to records *)
```

Comment:
Use example pattern, or else "braced identifier" is a bit difficult to understand for the comment reader.